src/libraries/Microsoft.PowerFx.Interpreter/Functions/StandardErrorHandling.cs

```diff
@@ -173,6 +173,99 @@ private static bool IsInvalidDouble(double number)
+                var arg1 = args[1];
+
+                KeyValuePair<IEnumerable<DValue<RecordValue>>, IEnumerable<DValue<RecordValue>>> allRows;
+                if (arg0 is TableValue both0 && arg1 is TableValue both1)
+                {
+                    if (both0.Rows.Count() == both1.Rows.Count())
+                    {
+                        allRows = new KeyValuePair<IEnumerable<DValue<RecordValue>>, IEnumerable<DValue<RecordValue>>>(both0.Rows, both1.Rows);
+                    }
+                    else
+                    {
+                        return CommonErrors.UnreachableCodeError(irContext);
+                    }
+                }
+                else if (arg0 is TableValue tv0)
+                {
+                    var inputRecordType = new RecordType().Add(BuiltinFunction.ColumnName_ValueStr, arg1.Type);
+                    var inputRecordNamedValue = new NamedValue(BuiltinFunction.ColumnName_ValueStr, arg1);
+                    var inputRecord = new InMemoryRecordValue(IRContext.NotInSource(inputRecordType), new List<NamedValue>() { inputRecordNamedValue });
+                    var inputDValue = DValue<RecordValue>.Of(inputRecord);
+                    var inputRows = tv0.Rows.Select(_ => inputDValue);
+                    allRows = new KeyValuePair<IEnumerable<DValue<RecordValue>>, IEnumerable<DValue<RecordValue>>>(tv0.Rows, inputRows);
+                }
+                else if (arg1 is TableValue tv1)
+                {
+                    var inputRecordType = new RecordType().Add(BuiltinFunction.ColumnName_ValueStr, arg0.Type);
+                    var inputRecordNamedValue = new NamedValue(BuiltinFunction.ColumnName_ValueStr, arg0);
+                    var inputRecord = new InMemoryRecordValue(IRContext.NotInSource(inputRecordType), new List<NamedValue>() { inputRecordNamedValue });
+                    var inputDValue = DValue<RecordValue>.Of(inputRecord);
+                    var inputRows = tv1.Rows.Select(_ => inputDValue);
+                    allRows = new KeyValuePair<IEnumerable<DValue<RecordValue>>, IEnumerable<DValue<RecordValue>>>(inputRows, tv1.Rows);
+                }
+                else
+                {
+                    return targetFunction(runner, symbolContext, irContext, (T[])args);
+                }
+
+                var tableType = (TableType)irContext.ResultType;
+                var resultType = tableType.ToRecord();
+                var itemType = resultType.GetFieldType(BuiltinFunction.OneColumnTableResultNameStr);
+
+                var zipped = allRows.Key.Zip(allRows.Value, (a, b) => new KeyValuePair<DValue<RecordValue>, DValue<RecordValue>>(a, b));
+                var resultRows = new List<DValue<RecordValue>>();
+                foreach (var pair in zipped)
+                {
+                    if (pair.Key.IsValue && pair.Value.IsValue)
+                    {
+                        var value0 = pair.Key.Value.GetField(BuiltinFunction.ColumnName_ValueStr);
```

Comment:
The record won't necessarily have a 'Value' property
![image](https://user-images.githubusercontent.com/44986255/142941094-1fd24274-2052-4adf-b1a8-0d4451d2aeb1.png)

Reply:
So what is the rule here? If the record only contains a single field, you just use the single field regardless of the name? If the record has more than 1 field, does it search for the "Value" field?

Reply:
It needs to be a single-column table; if there is more than one column, the compilation will fail - see https://github.com/microsoft/Power-Fx/blob/main/src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Concatenate.cs#L60

Reply:
Added support for other single column names for the input table(s).